module/Application/view/layout/layout.phtml

```diff
@@ -22,30 +22,28 @@
                             ])
             ->prependStylesheet($this->basePath('css/style.css'))
             ->prependStylesheet($this->basePath('css/bootstrap.min.css'))
-?>
-
+        ?>
         <!-- Scripts -->
         <?= $this->headScript() ?>
     </head>
     <body>
-        <nav class="navbar navbar-expand-md navbar-dark mb-4" role="navigation">
+        <nav class="navbar navbar-expand-sm navbar-dark mb-4" role="navigation">
             <div class="container">
-                <div class="navbar-header">
-                    <button
-                        class="navbar-toggler"
-                        type="button"
-                        data-bs-toggle="collapse"
-                        data-bs-target="#navbarSupportedContent"
-                        aria-controls="navbarSupportedContent"
-                        aria-expanded="false"
-                        aria-label="Toggle navigation"
-                    >
-                        <span class="navbar-toggler-icon"></span>
-                    </button>
-                    <a class="navbar-brand" href="<?= $this->url('home') ?>">
-                        <img src="<?= $this->basePath('img/laminas-logo.svg') ?>" alt="Laminas">MVC Skeleton
-                    </a>
-                </div>
+                <a class="navbar-brand" href="<?= $this->url('home') ?>">
+                    <img src="<?= $this->basePath('img/laminas-logo.svg') ?>" alt="Laminas">
+                    <span class="navbar-text text-light">MVC Skeleton</span>
+                </a>
+                <button
+                    class="navbar-toggler"
+                    type="button"
+                    data-bs-toggle="collapse"
+                    data-bs-target="#navbarSupportedContent"
+                    aria-controls="navbarSupportedContent"
+                    aria-expanded="false"
+                    aria-label="Toggle navigation"
+                >
+                    <span class="navbar-toggler-icon"></span>
+                </button>
                 <div class="collapse navbar-collapse" id="navbarSupportedContent">
                     <ul class="navbar-nav">
```

Comment:
Have you tested if we can place the navigation on the right side as well?
```suggestion
                    <ul class="navbar-nav ms-auto">
```

Reply:
With only using the margin start auto if the user then adds an additional menu entry they are going to start stacking up vertically. Another issue with simply pushing the menu right is that once its toggled it will then align left when the toggle is expanded putting it under the logo. Im sure a solution could be found to push them both right but Im pretty certain it will mean expanding the amount of markup that is needed to accomplish it. It seemed more natural, to me at least, that both the main nav and the toggled nav be left aligned so its not jumping back and forth. Not to mention I really was not trying to add additional markup just to accomplish it.

Reply:
After some more testing I decided to bump the breakpoint back to medium instead of on the small.

Reason:
Although the small breakpoint is better for a single menu item (home) if the user adds entries for example
Home (default)
About
Test
Test Two
Then before the small breakpoint is reached the test two entry will wrap and it screws up the vertical alignment. So with so few additions required to break it I thought is best to move back to the md breakpoint.

1. The menu is now right aligned when above the collapse point. 
2. The collapse breakpoint is now back to medium so that more items can be accomodated and preventing vertical alignment issues.
3. Once the menu collapses the menu is of course left aligned. I have been looking into a solution to that. I am undecided about that point personally. It will most likely take additional styles to facilitate the moving of the menu items to the right once its collapsed.